modules/nodes-custom-scheduler-packaging.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-custom-scheduler.adoc
+
+[id="nodes-custom-scheduler-packaging_{context}"]
+= Packaging and deploying a custom scheduler
+
+To include a custom scheduler in your cluster, create an image and include that image in a deployment.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a scheduler binary.
++
+[NOTE]
+====
+Information on how to create the scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation.
+====
```

Comment:
Can we wrap this in a support statement? We support the capability to swap out schedulers but don't directly support custom schedulers as they are replacements to the system/platform offering/design? 

We may want to run this by pm? Cc: @reestr 

I'm ok with the link if we remove the statement about out of scope and replace it with a support statement.